ros_ws/src/navigation_stack/car_cartographer/config/transform.lua

```diff
@@ -0,0 +1,28 @@
+-- Copyright 2017 The Cartographer Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+XY_TRANSFORM =  {
+    translation = { 0., 0., 0. },
+    rotation = { 0., -math.pi / 2., 0., },
+  }
+  
+  XZ_TRANSFORM =  {
+    translation = { 0., 0., 0. },
+    rotation = { 0. , 0., -math.pi / 2, },
+  }
+  
+  YZ_TRANSFORM =  {
+    translation = { 0., 0., 0. },
+    rotation = { 0. , 0., math.pi, },
+  }
```

Comment:
This file could be moved into `cartographer_config_assets_writer.lua` if it's not used anywhere else.

Reply:
Done.